addons/website/static/src/snippets/s_embed_code/options.js

```diff
@@ -7,6 +7,14 @@ import options from 'web_editor.snippets.options';
 const _t = core._t;
 
 options.registry.EmbedCode = options.Class.extend({
+    /**
+     * @override
+     */
+     start() {
+        this.$target.on('click', '.s_embed_code_edit', () => this.editCode());
```

Comment:
You should `off` that in the `destroy` method. Basically, everytime you `on` something, there should be a `off` somewhere, even if you think of no way where it is needed, it could be. In this case, there are ways (although the flows I think about seems buggy :grimacing:) but:
- Add a "code" snippet
- Play enough with the page: add snippets, inline snippets in them, remove some, etc
- Click the "edit" button in your "code" snippet: the modal opens multiple times.

(normally I think all options are destroyed each time we add/move a snippet, that seems not the case anymore, may be a bug. That was not strictly needed just necessary to preview some options correctly, I'll investigate: but the point here is: what is initialized should be destroyed :wink:)

Reply:
Hm, indeed, I always forget about that and end up copy pasting wrong code.
There is some options not cleaning their event like s_image_gallery, from where I got inspired :grimacing: